### PR TITLE
Regular Release 11

### PR DESCRIPTION
**Pull request by mamoonraja**
Ruby SDK regular release # 11 includes:

- Compare and Comply release 
- Update to NLC and Visual recognition create and update classifier methods, proposed in https://github.com/watson-developer-cloud/ruby-sdk/pull/34
- Changes to services because of swagger updates
- Refactoring for personality insights
- minor tweaks to `rubocop` because of change from `ArgumentError` to `ArgumentError.new`

### CODE REVIEW

File: lib/ibm_watson/natural_language_understanding_v1.rb [hunk 1]
```diff
@@ -96,22 +96,33 @@ def initialize(args = {})
...
+    # @param url [String] The webpage to analyze. One of the `text`, `html`, or `url` parameters is
     #   required.
-    # @param clean [Boolean] Remove website elements, such as links, ads, etc.
-    # @param xpath [String] An [XPath query](https://www.w3.org/TR/xpath/) to perform on `html` or `url`
-    #   input. Results of the query will be appended to the cleaned webpage text before it
-    #   is analyzed. To analyze only the results of the XPath query, set the `clean`
-    #   parameter to `false`.
+    # @param clean [Boolean] Set this to `false` to disable webpage cleaning. To learn more about webpage
+    #   cleaning, see the [Analyzing
+    #   webpages](/docs/services/natural-language-understanding/analyzing-webpages.html)
```
Comment (mediumTaj) [inline lib/ibm_watson/natural_language_understanding_v1.rb]: Mamoon can you refactor these relative urls to use 
https://cloud.ibm.com/docs/services/...

File: lib/ibm_watson/speech_to_text_v1.rb [hunk 2]
```diff
@@ -361,15 +361,14 @@ def recognize(audio:, content_type: nil, model: nil, language_customization_id:
     end
 
     ##
-    # @!method recognize_using_websocket(content_type: nil,recognize_callback:,audio: nil,chunk_data: false,model: nil,customization_id: nil,language_customization_id: nil,acoustic_customization_id: nil,customization_weight: nil,base_model_version: nil,inactivity_timeout: nil,interim_results: nil,keywords: nil,keywords_threshold: nil,max_alternatives: nil,word_alternatives_threshold: nil,word_confidence: nil,timestamps: nil,profanity_filter: nil,smart_formatting: nil,speaker_labels: nil)
+    # @!method recognize_using_websocket(content_type:,recognize_callback:,audio: nil,chunk_data: false,model: nil,customization_id: nil,acoustic_customization_id: nil,customization_weight: nil,base_model_version: nil,inactivity_timeout: nil,interim_results: nil,keywords: nil,keywords_threshold: nil,max_alternatives: nil,word_alternatives_threshold: nil,word_confidence: nil,timestamps: nil,profanity_filter: nil,smart_formatting: nil,speaker_labels: nil)
```
Comment (mediumTaj) [inline lib/ibm_watson/speech_to_text_v1.rb]: Content type here should be optional

File: lib/ibm_watson/tone_analyzer_v3.rb [hunk 3]
```diff
@@ -97,7 +97,7 @@ def initialize(args = {})
     #########################
 
     ##
-    # @!method tone(tone_input:, content_type:, sentences: nil, tones: nil, content_language: nil, accept_language: nil)
+    # @!method tone(tone_input:, content_type: nil, sentences: nil, tones: nil, content_language: nil, accept_language: nil)
```
Comment (mediumTaj) [inline lib/ibm_watson/tone_analyzer_v3.rb]: Can you check to make sure content type can be optional here?

File: lib/ibm_watson/speech_to_text_v1.rb (hunk 2 above)
Comment (mamoonraja) [reply lib/ibm_watson/speech_to_text_v1.rb]: Good catch! This change needs to go to post generation of STT.

File: lib/ibm_watson/tone_analyzer_v3.rb (hunk 3 above)
Comment (mamoonraja) [reply lib/ibm_watson/tone_analyzer_v3.rb]: Looks like it is optional.

File: lib/ibm_watson/natural_language_understanding_v1.rb (hunk 1 above)
Comment (mamoonraja) [reply lib/ibm_watson/natural_language_understanding_v1.rb]: done

**Pull request merged**